### PR TITLE
Theme Selecting Options

### PR DESCRIPTION
**Pull request by parimatrix**
Provide a general summary of your changes in the **Title** above.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### Motivation and Context
- [x] Why is this change required? What problem does it solve?
        Design changes. Allows to choose between two themes
- [ ] If it fixes an open issue, include the text `Closes #1` (where 1 would be the issue number) to your commit message.

### Types of changes
What types of changes does your code introduce? Check all the boxes that apply:
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Description
- [x ] Describe your changes in detail.
       The changes are not final. The theme selector buttons can be styled too. Also the player has not 
        been styled till now.
### Final checklist:
Go over all the following points and check all the boxes that apply  
If you're unsure about any of these, don't hesitate to ask. We're here to help!
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](.github/CONTRIBUTING.md) guidelines.
- [x] All tests passed.

### CODE REVIEW

**Comment by shakeelmohamed** [comment]
Thanks for the efforts @parimatrix! I'd rather see these styles changed dynamically with [jQuery's CSS API](http://api.jquery.com/css/) Also note that there are some code climate issues with your changes, click details for more info

**Comment by parimatrix** [comment]
ok. Will try to use the API. And sorry for the code-climate issues , did not know about them.

**Comment by shakeelmohamed** [comment]
@parimatrix no worries, it's all part of the learning experience 👍

**Comment by parimatrix** [comment]
after researching a bit on the net, i found that jQuery dynamic changes are less efficient if many changes are to be made in the file. So I have changed the file structure to prevent duplication of data. any feedback would be great

File: index.html [hunk 1]
```diff
@@ -30,6 +32,8 @@
 
 </head>
 <body>
+    <button id="themeselector1">Basic Theme</button>
```
Comment (shakeelmohamed) [inline index.html]: The id on both of these buttons should be something like `themeSelectorBasic` so it reflects the theme name

File: index.html [hunk 2]
```diff
@@ -68,8 +72,9 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
             </h3>
 
             <div id="playerControls">
-                <button class="btn" id="togglePlayer">
+                <button class="mybtn" id="togglePlayer">
```
Comment (shakeelmohamed) [inline index.html]: Why rename this class?

File: index.html [hunk 3]
```diff
@@ -40,18 +44,18 @@
...
+            <h2 class="">Listen to YouTube videos, without the distracting visuals.</h2>
             <h2 class="color-grey hide flash" id="mobile-message"></h2>
         </hgroup>
     </header>
     <div id="container">
         <!-- The form -->
         <form id="form">
-            <div class="input-group">
-                <input type="text" id="v" name="v" class="input-search color-grey" placeholder="Search, YouTube video ID, or URL">
+            <div id = "shadow" class="input-group">
```
Comment (shakeelmohamed) [inline index.html]: nit:  remove the spaces in `id = "shadow"`

File: index.html [hunk 4]
```diff
@@ -128,5 +133,20 @@ <h2 class="color-grey hide flash" id="mobile-message"></h2>
             "logo": "https://zenplayer.audio/img/zen-audio-player-905.png"
         }
     </script>
+    <script>
+    var basic = document.getElementById("themeselector1");
```
Comment (shakeelmohamed) [inline index.html]: Let's use jQuery click handlers for consistency, and they can be moved to `js/everything.js`

File: css/style1.css [hunk 5]
```diff
@@ -0,0 +1,35 @@
+body {
```
Comment (shakeelmohamed) [inline css/style1.css]: Let's rename these CSS files to match the style name

File: index.html [hunk 6]
```diff
@@ -11,11 +11,13 @@
 
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
+    <link href="https://fonts.googleapis.com/css?family=Open+Sans" rel="stylesheet">
```
Comment (shakeelmohamed) [inline index.html]: let's move this down a with the other stylesheets.
nit: put the `rel` attribute first to match the others

File: index.html [hunk 7]
```diff
@@ -11,11 +11,13 @@
...
 
     <link rel="stylesheet" href="bower_components/primer-css/css/primer.css">
     <link rel="stylesheet" href="bower_components/font-awesome/css/font-awesome.min.css">
     <link rel="stylesheet" href="bower_components/plyr/dist/plyr.css">
     <link rel="stylesheet" href="css/styles.css">
+    <link rel="stylesheet" id= "cssselector" href="css/style1.css">
```
Comment (shakeelmohamed) [inline index.html]: nit: remove the space after `id=`

Comment (shakeelmohamed) [inline index.html]: probably should change the `id` to something more descriptive like `currentTheme`

File: index.html [hunk 8]
```diff
@@ -40,18 +44,18 @@
 
         <hgroup>
             <h1>Zen Audio Player</h1>
-            <h2 class="color-grey">Listen to YouTube videos, without the distracting visuals.</h2>
+            <h2 class="">Listen to YouTube videos, without the distracting visuals.</h2>
```
Comment (shakeelmohamed) [inline index.html]: nit: remove the `class` attribute if it's empty

**Comment by parimatrix** [comment]
Thanks a lot for the detailed feedback. Am working on it. Will finish soon. 👍

**Comment by parimatrix** [comment]
@shakeelmohamed  I tried to fix all the design and structure issues. Only one error is left in Travis build. But I am not able to understand the error message. Could you please review. Thanks

File: index.html [hunk 9]
```diff
@@ -11,11 +11,13 @@
 
     <link rel="shortcut icon" href="img/favicon.ico" type="image/x-icon">
     <link rel="icon" href="img/favicon.ico" type="image/x-icon">
-
+    
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Open+Sans">
```
Comment (shakeelmohamed) [inline index.html]: If you move this tag down after `basic.css` the test should pass

**Comment by parimatrix** [comment]
Thanks a lot for the help.

File: index.html (hunk 2 above)
Comment (parimatrix) [inline index.html]: btn is a predefined class of bootstrap. I wished to overwrite some of its properties. Using !important was not a great option, so designed the button afresh

**Comment by parimatrix** [comment]
Hi. Any further changes to be made that would help these designs to merged ?

**Comment by shakeelmohamed** [comment]
@parimatrix sorry, I haven't had a chance to look at this again. I will let you know when I do

**Comment by shakeelmohamed** [comment]
@parimatrix I tried testing your changes locally. The buttons have no effect, and they look very out of place - see the screenshot below.
capturfiles2

**Comment by parimatrix** [comment]
Sorry for the delayed reply. I will look into it. Also will style the buttons.

**Comment by shakeelmohamed** [comment]
Closing due to inactivity

**Pull request closed without merging**